src/commands.cpp

```diff
@@ -4503,12 +4498,14 @@ void CommandChatCreate::procresult()
                         TextChat *chat = client->chats[chatid];
                         chat->id = chatid;
                         chat->priv = PRIV_MODERATOR;
-                        chat->url = url;
                         chat->shard = shard;
                         delete chat->userpriv;  // discard any existing `userpriv`
                         chat->userpriv = this->chatPeers;
                         chat->group = group;
 
+                        chat->setTag(tag ? tag : -1);
```

Comment:
Exactly. Currently, the `tag` is never set to `0` since the SDK does not generate chat-related commands by itself. However, if one day we start doing it, we are already protected.